sdk/storage/azure-storage-blob/swagger/Update-Codegeneration.ps1

```diff
@@ -0,0 +1,8 @@
+npm install -s -g autorest
```

Comment:
Why duplicate these scripts instead of having one common one that takes a parameter?

Reply:
If you want to essentially default the path then I'd suggest calling into a common version of the script passing in the PSScriptRoot as a parameter. I suspect the actually generation steps may change overtime and it would be annoying to have to update all these.

Reply:
That's a good point, I do want a script to live in each folder where there is a Swagger/code gen configuration so there can be external automation to run all code generation in a specific folder or in the whole repo but there's no reason that each of these needs to contain all the setup logic.

I'll make another change to add a shared script which these call into.